integration-cli/docker_cli_cp_test.go

```diff
@@ -596,3 +596,82 @@ func (s *DockerSuite) TestCpNameHasColon(c *check.C) {
 		c.Fatalf("Wrong content in copied file %q, should be %q", content, "lololol\n")
 	}
 }
+
+func (s *DockerSuite) TestCpGlob(c *check.C) {
+	command := "mkdir '/other' && mkdir -p '/some/path/other' && echo -n '" + cpContainerContents + "' > " + cpFullPath + " && echo -n '" + cpContainerContents + "' > /some/path/other/test"
+
+	out, exitCode := dockerCmd(c, "run", "-d", "busybox", "/bin/sh", "-c", command)
+	if exitCode != 0 {
+		c.Fatal("failed to create a container", out)
+	}
+
+	cleanedContainerID := strings.TrimSpace(out)
+
+	out, _ = dockerCmd(c, "wait", cleanedContainerID)
+	if strings.TrimSpace(out) != "0" {
+		c.Fatal("failed to set up container", out)
+	}
+
+	if err := os.MkdirAll(cpTestPath, os.ModeDir); err != nil {
```

Comment:
Is the stuff on lines 615 thru 619 needed?  I'm not sure why we would need to create these dirs on the local machine if we cp into "tmpdir" ?